Framework/include/Baseline.h

```diff
@@ -39,9 +39,9 @@ class Baseline
         // -- Data dependent stuff
         // ------------------------------
         
-        bool passTriggerAllHad   = PassTriggerAllHad2016(TriggerNames, TriggerPass) || PassTriggerAllHad2017(TriggerNames, TriggerPass);
-        bool passTriggerMuon     = PassTriggerMuon(TriggerNames, TriggerPass);
-        bool passTriggerElectron = PassTriggerElectron(TriggerNames, TriggerPass);
+        bool passTriggerAllHad   = filetag.find("2017") ? PassTriggerAllHad2017(TriggerNames, TriggerPass) : PassTriggerAllHad2016(TriggerNames, TriggerPass);
```

Comment:
Should this line be 
bool passTriggerAllHad = filetag.find("2017") != std::string::npos ? PassTriggerAllHad2017(TriggerNames, TriggerPass) : PassTriggerAllHad2016(TriggerNames, TriggerPass);

Reply:
Ahh, good catch. Yes